ocd/src/collection/targets/mod.rs

```diff
@@ -1 +1 @@
-pub(crate) mod nrf51822;
\ No newline at end of file
+// pub(crate) mod nrf51822;
```

Comment:
Do you still need this file?

Reply:
I just left it for good measure before and then forgot to remove it. Will do.